api/src/main/java/org/apache/iceberg/Snapshot.java

```diff
@@ -195,4 +195,13 @@ default Long firstRowId() {
   default Long addedRows() {
     return null;
   }
+
+  /**
+   * ID of the encryption key used to encrypt this snapshot's manifest list.
+   *
+   * @return a string key ID
+   */
+  default String keyId() {
```

Comment:
would it make sense to name this `encryptionKeyId` as `keyId` might not be immediately obvious that it relates to encryption when browsing/reading the code

Reply:
didn't we call this encryptingKeyId in the spec proposal?

Reply:
I don't think that this is unclear and needs to be longer, but I'm open to making it `encryptionKeyId` if others think that it is.

Reply:
I don't want to change it unless we change it in the spec as well